components/Buttons/src/MDCButton.m

```diff
@@ -520,12 +539,18 @@ - (void)setBackgroundColor:(UIColor *)backgroundColor forState:(UIControlState)s
 
 - (CGFloat)elevationForState:(UIControlState)state {
```

Comment:
There were some tests checking elevation was getting set. can we bring those into this PR?